src/components/vote-form.js

```diff
@@ -1,14 +1,121 @@
-import React from "react";
-import styled from "styled-components";
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 
-export default function VoteForm() {
-  return <Wrapper>안녕 나는 투표 폼!</Wrapper>;
+import styled from 'styled-components';
+
+function VoteForm() {
+	const [candi, setCandi] = useState([]);
+	const [voteCount, setVoteCount] = useState();
+
+	useEffect(() => {
+		getCandidates();
+	}, []);
+
+	const getCandidates = async () => {
+		await axios
+			.get(process.env.API_HOST + '/candidates/', candi)
+			.then(({ data }) => {
+				setCandi(data);
+			})
+			.catch(function (error) {
+				console.log(error);
+			});
+	};
+
+	const voteCandidates = async (person) => {
+		const newUrl =
+			process.env.API_HOST + '/candidates/' + person._id + '/vote/';
+		await axios
+			.put(newUrl, voteCount)
+			.then(({ data }) => {
+				setVoteCount(data);
+				alert(person.name + '님에게 투표 완료!');
+				getCandidates();
+			})
+			.catch(function (error) {
+				console.log(error);
+				alert('투표 실패!');
+			});
+	};
+	let i = 1;
+	return (
+		<Wrapper>
+			<Title>
+				<RedTitle>프론트엔드 인기쟁이</RedTitle>는 누구?
+			</Title>
+			<SubTitle>CEOS 프론트엔드 개발자 인기 순위 및 투표 창입니다.</SubTitle>
+			<VoteArea>
+				{candi
+					.sort((person1, person2) => {
+						return person2.voteCount - person1.voteCount;
+					})
+					.map((person) => (
+						<Row key={JSON.stringify(person)}>
+							<Rank>{i++}위:</Rank>
```

Comment:
1. JSON.stringify 사용해주신 것 정말 좋네요! 그런데 이 상황에선 id가 넘어오기 때문에 그 값을 key로 이용하면 될 것 같습니다 ㅎ
2. i 라는 별도의 변수를 사용하는 것보다 map 함수의 index를 사용하는게 더 좋을 것 같아요!
```suggestion
					.map((person, index) => (
						<Row key={person._id}>
							<Rank>{index + 1}위:</Rank>
```